implementations/rust/ockam/ockam_transport_tcp/src/portal/mod.rs

```diff
@@ -6,6 +6,8 @@ mod portal_worker;
 
 pub(crate) use inlet_listener::*;
 pub(crate) use outlet_listener::*;
+pub use portal_message::PortalMessage;
```

Comment:
We would probably want to move Portals protocol-related things out of Tcp, probably to ockam_core, or ockam_transport_core. This is for the future